Library/Homebrew/cmd/upgrade.rb

```diff
@@ -191,8 +194,8 @@ def upgrade_outdated_formulae(formulae, args:)
       end
     end
 
-    if !pinned.empty? && !args.ignore_pinned?
-      ofail "Not upgrading #{pinned.count} pinned #{Utils.pluralize("package", pinned.count)}:"
+    unless pinned.empty?
```

Comment:
Small nit but maybe:

```suggestion
    if pinned.any?
```